azure-pipelines.yml

```diff
@@ -35,10 +35,16 @@ stages:
         Windows:
           imageName: 'windows-latest'
     steps:
-    - task: NodeTool@0
+    - task: npmAuthenticate@0
```

Comment:
I don't think this is required since the registry allows for anonymous access.

Reply:
Thank you for reviewing. I agree with you but if we don't use npmAuthenticate after using the new registry we get `NPM ERR Code E401: Unable to authenticate` on pipeline machines which is ... weird, but if it works it works I suppose.

Reply:
That generally means that required packages have not been ingested from the upstream. I'm pretty surprised it worked with the step, because the public public can't ingest.

Reply:
Good point, thanks for the help there. The packages were not ingested upstream at the time the PR was created. So we were hoping we could remove the npmauth, as we checked and at least a few of the packages did get ingested upstream, such as the vscode npm package. I tried removing the npmauth and we still get 401 unauthorized errors again. 

cc @rainersigwald

Reply:
We were able to get the upstream packages in the feed by running `npm install -g tslint` on my local machine (with auth + permission to add to the feed) once; now public restores of that step are working.